src/Symfony/Component/Workflow/Tests/Validator/StateMachineValidatorTest.php

```diff
@@ -116,12 +116,15 @@ public function testValid()
 
     public function testWithTooManyInitialPlaces()
     {
-        $this->expectException(InvalidDefinitionException::class);
-        $this->expectExceptionMessage('The state machine "foo" cannot store many places. But the definition has 2 initial places. Only one is supported.');
+
         $places = range('a', 'c');
         $transitions = [];
         $definition = new Definition($places, $transitions, ['a', 'b']);
 
+
+        $this->expectException(InvalidDefinitionException::class);
+        $this->expectExceptionMessage('The state machine "foo" cannot store many places. But the definition has 2 initial places. Only one is supported.');
+
         (new StateMachineValidator())->validate($definition, 'foo');
 
         // the test ensures that the validation does not fail (i.e. it does not throw any exceptions)
```

Comment:
are the following lines still needed @lyrixx I don't understand that part 🤔

Reply:
this makes no sense I agree, let's see what happens when removing the next lines

Reply:
could be also the result of a bad merge ? how does this look like on lower branches?

Reply:
The code looks the same until 4.4, I will provide a PR against 5.4 for this test and revert here